### ISSUE TITLE
use of undeclared identifier HUGE_VAL on mac osx

### ISSUE THREAD

**Issue by ofttryaj** [zchee/deoplete-go#150]
When I make deopoete-go, it does not work
python3.7.1
use of undeclared identifier HUGE_VAL on mac osx

**Comment by Shougo** [comment]
I have searched in the source code, and I have not found `HUGE_VAL`.
I think it is not deoplete-go's issue.

**Comment by ofttryaj** [comment]
Yes, it is deoplete-go/rplugin/python3/deoplete/ujson @ 2f1d487 issue

**Comment by Shougo** [comment]
Hm.  If so, you can fix it.

https://github.com/esnme/ultrajson/tree/2f1d4874f4f4d2a40a460678004c80e69387c663
